FitNesseRoot/DbFit/AcceptanceTests/JavaTests/SqlServerTests/properties.xml

```diff
@@ -2,13 +2,14 @@
 <properties>
 	<Edit/>
 	<Files/>
-	<LastModified>20080330043718</LastModified>
 	<Properties/>
 	<RecentChanges/>
 	<Refactor/>
 	<Search/>
 	<Suite/>
-	<Suites/>
+	<SymbolicLinks>
+		<DisabledTests>.DbFit.AcceptanceTests.DotNetTests.SqlServerTests.DisabledTests</DisabledTests>
```

Comment:
This is so the `DisabledTests` suite is visible under the `JavaTests` suite else they're invisible unless you visit the `DotNetTests` suite.

Do you think this is the wrong thing to do? I thought I would be another reminder to get them working some time!